scripts/manage/landing_page_secured.md

```diff
@@ -22,6 +22,15 @@ View the who, what, when and where of your Spinnaker installation
 View the logging output of the individual components of your Spinnaker installation
 [here](https://console.developers.google.com/logs/viewer?project={{project-id}}&resource=k8s_container%2Fcluster_name%2F$GKE_CLUSTER%2Fnamespace_name%2Fspinnaker).
 
+### Try out Spinnaker samples
```

Comment:
```suggestion
### Install sample applications and pipelines
```

Reply:
/cc @dorbin on this. Depending on how we feel about this change, it would also need to be made below

Reply:
Done